core/metacling/src/TClingValue.cxx

```diff
@@ -44,6 +44,25 @@ TClingValue& TClingValue::operator=(TClingValue &Other) {
    return *this;
 }
 
+std::pair<std::string, std::string> TClingValue::ToTypeAndValueString() const {
+  std::string output = ToString();
+  int pos, bracket = 0, str_size = output.size() - 1;
+  for (pos = 0; pos <= str_size; pos++) {
+    if (output[pos] == '(')
+      bracket++;
```

Comment:
In general (not for `int`s), `--var` is cheaper than `var--`. It's a good habit to use prefix wherever possible.